examples/example_morph.m

```diff
@@ -28,7 +28,7 @@
 % Alternative way to add 3D point to main using dynamic top-level function.
 main.push();
 n.pt3dadd(201, 2, 3, 1);
-main.pop();
+neuron.stack.pop_sections(1);
```

Comment:
Is this going to be a problem in the merge since we changed our to `n.pop_section()` in the last pull request?

Reply:
No, I will do a rebase before changing the PR from 'draft' to 'ready'.